### ISSUE TITLE
Compilation fail: No such file or directory: u'/diracos_repo/x86_64'

### ISSUE THREAD

**Issue by petricm** [DIRACGrid/DIRACOS#17]
I get this error when trying to build the OS
```sh
docker run --privileged --cap-add=SYS_ADMIN --oom-kill-disable=true -it cern/slc6-base bash

yum install -y mock rpm-build fedora-packager createrepo python-pip

export DIRACOS_REPO=/cvmfs/clicdp.cern.ch/DIRAC/OS
mkdir -p $DIRACOS_REPO/i386 $DIRACOS_REPO/i686 $DIRACOS_REPO/src $DIRACOS_REPO/noarch $DIRACOS_REPO/x86_64 $DIRACOS_REPO/buildOnly
createrepo $DIRACOS_REPO  

pip install git+https://github.com/DIRACGrid/DIRACOS.git
git clone https://github.com/DIRACGrid/DIRACOS.git

dos-build-all-rpms DIRACOS/config/diracos.json >& log.txt
```

[log.txt](https://github.com/DIRACGrid/DIRACOS/files/2160488/log.txt)

**Comment by chaen** [comment]
Did you update the value here ? 
https://github.com/DIRACGrid/DIRACOS/blob/master/mockConfigs/mock-build-diracos.cfg#L92

I think I mention that it is needed to update it several times (like https://github.com/DIRACGrid/DIRACOS#configuration-files) but maybe it is not clear enough

**Comment by petricm** [comment]
Sorry, was doing this on the airport before boarding, I retried now but have to board again. But I would say that this is not enough clear in the docs, I would at least add a link to #L92 ... Tell you more once I get to the end with the compilation.

**Comment by chaen** [comment]
OK, point taken, waiting for more feedback before correcting the doc :) Thanks for trying !

**Comment by petricm** [comment]
This is considered resolved, but we let it open until documentation is mentioning all the necessary steps to build in a custom directory.

**Comment by petricm** [comment]
Resolved in #21
